docs/public/static/icons/shutterstock.svg

```diff
@@ -0,0 +1,15 @@
+<path d="M33.8031 17.2548C33.2348 17.5605 32.5157 17.7358 31.8002 17.7358C29.569 17.7358 28.7303 16.6022 28.7303 15.0387V6.02849H25.7412V15.0936C25.7412 18.6173 27.783 20.313 31.4323 20.313C33.1799 20.313 35.1153 19.8722 36.7911 19.1486V6.02849H33.8031V17.2548Z" fill="#8796A5"/>
+<path d="M7.30393 11.7372C7.09692 11.7006 6.39599 11.5846 6.19039 11.5471C5.08709 11.3488 4.13817 11.091 4.13817 9.98149C4.13817 9.02886 5.13848 8.58441 6.38598 8.58441C8.23976 8.58441 9.86583 9.42248 10.0514 9.51498L11.0672 6.84057C10.6788 6.63817 9.02279 5.7566 6.41586 5.7566C4.03128 5.7566 1.11615 7.00643 1.11615 10.1159C1.11615 12.8986 3.10755 13.7917 5.03139 14.1289C5.29616 14.1776 6.13079 14.3197 6.37083 14.3612C7.6625 14.5791 8.36919 15.0591 8.36919 15.9024C8.36919 16.9729 7.36417 17.4748 5.8388 17.4748C3.39322 17.4748 1.9596 16.7484 1.54494 16.5785L0.419922 19.1951C0.770697 19.3749 2.43816 20.313 5.69302 20.313C8.85142 20.313 11.3861 18.7527 11.3861 15.8487C11.3589 13.1455 9.53762 12.12 7.30393 11.7372Z" fill="#8796A5"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M61.2338 5.7556C64.7554 5.7556 66.7139 8.00868 66.7139 10.5718V14.3867L58.684 14.3867L58.6944 15.1543C58.6944 16.4431 59.4053 17.4717 61.2537 17.4717C62.6172 17.4717 63.6163 16.5979 63.8318 15.9752L66.3553 17.4874C65.8836 18.5477 64.2947 20.313 61.2338 20.313C57.764 20.313 55.7012 18.4458 55.7012 15.226V10.5718C55.7012 8.24559 57.2361 5.7556 61.2338 5.7556ZM58.684 11.9725L63.7243 11.9725V10.8976C63.7243 9.78379 63.112 8.59587 61.2196 8.59587C59.2564 8.59587 58.684 9.81563 58.684 10.9529V11.9725Z" fill="#8796A5"/>
+<path d="M68.627 10.5771V20.0415H71.6139V10.8317C71.6139 10.0274 72.0716 8.65282 74.2654 8.65282C74.6098 8.65282 75.3883 8.75065 75.6373 8.79724V5.90668C75.3679 5.84173 74.7503 5.75556 74.1124 5.75556C70.3128 5.75556 68.627 8.13261 68.627 10.5771Z" fill="#8796A5"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M135.36 7.11609C135.36 7.92247 136.015 8.53418 136.823 8.53418C137.632 8.53418 138.28 7.92247 138.28 7.11609C138.28 6.31666 137.624 5.66307 136.823 5.66307C136.015 5.66307 135.36 6.30992 135.36 7.11609ZM136.823 5.98987C136.19 5.98987 135.729 6.48398 135.729 7.11609C135.729 7.70699 136.196 8.20738 136.823 8.20738C137.457 8.20738 137.911 7.70699 137.911 7.11609C137.911 6.49031 137.416 5.98987 136.823 5.98987ZM137.018 7.80425C137.026 7.81789 137.039 7.82464 137.054 7.82464H137.381C137.395 7.82464 137.402 7.81094 137.395 7.79687L137.116 7.3038C137.304 7.2203 137.408 7.0397 137.408 6.84487V6.81711C137.408 6.59525 137.241 6.35855 136.865 6.35855H136.308V7.79687C136.308 7.81094 136.322 7.82464 136.335 7.82464H136.615C136.628 7.82464 136.636 7.81094 136.636 7.79687V7.3175H136.795L137.018 7.80425ZM136.823 6.60895C137.011 6.60895 137.081 6.7129 137.081 6.82385V6.84487C137.081 6.9907 137.026 7.08116 136.823 7.08116H136.636V6.60895H136.823Z" fill="#8796A5"/>
+</svg>
```

Comment:
- This icons folder is for the logo. Nothing else. Could we use the branding folder to isolate these assets?
- Also note that we already have https://github.com/mui-org/material-ui/blob/next/docs/public/static/branding/pricing/shutterstock.svg.
- Also note that these SVGs can be compressed with https://jakearchibald.github.io/svgomg/.

Reply:
> Note that we already have company logos

Okay I can reuse those.